### ISSUE TITLE
Running "mw -a mail@example.com" wants init of pass. Store is already there

### ISSUE THREAD

**Issue by JohnnyLeone** [LukeSmithxyz/mutt-wizard#728]
Hi,

thanks for this awesome project.

just tried to add a new mail Address but i always get the first run message to initialize the pass store. I tested the function [checkbasics](https://github.com/LukeSmithxyz/mutt-wizard/blob/b0452a39d5b573a9a12da0064c0341846ba8ff5d/bin/mw#L19) and found that the gpg command returns in my case of a multiline .gpg-id always "gpg: error reading key: No secret key"

If i remove the double quote of the [$(cat "$PASSWORD_STORE_DIR/.gpg-id")](https://github.com/LukeSmithxyz/mutt-wizard/blob/b0452a39d5b573a9a12da0064c0341846ba8ff5d/bin/mw#L22) it works fine. The newline seems ignored with the current approach. 

.gpg-id looks like this:

```
0x6133FC54EC90C0E4!
0xDC60C96F3502AB2A!
```

**Different output of both functions:**

Don't work:
```
printf "<%s>\n" "$GPG" --list-secret-keys "$(cat "$PASSWORD_STORE_DIR/.gpg-id")"
<gpg>
<--list-secret-keys>
<0x6133FC54EC90C0E4!
0xDC60C96F3502AB2A!>
```
Works:
```
printf "<%s>\n" "$GPG" --list-secret-keys $(cat "$PASSWORD_STORE_DIR/.gpg-id")
<gpg>
<--list-secret-keys>
<0x6133FC54EC90C0E4!>
<0xDC60C96F3502AB2A!>
```

**Comment by JohnnyLeone** [comment]
Hi @Kr1ss-XD
sorry for long time since my report. I need to use multiple IDs because for some pass files. I encrypt them with multiple subkeys of my gpg key. This different key is stored on a portable USB gpg card.
With this second .gpg-id entry, every file i create or edit in this store will be encrypted for both keys.

**Comment by Zocker1999NET** [comment]
I ran into a similar issue, my use case is still a bit different: My main GPG key is also stored on smart cards. As fallback, if it happens that all smart cards will be destroyed, I created an "archive" key which happens to be better secured against physical danger and which is not used daily. With this key, I can ensure that I will still be able to access certain data even if  all smart cards get lost.
